### PR TITLE
Documentation has been added for Electronic Gadget site

### LINKED ISSUE #2514: Adding Readmefile to Electronic Gadget website
**Describe the Feature you want to end** 
Adding readme file to project site

**Expected behavior**
helps in better understanding

**Screenshots (optional)**

**Additional context (optional )**
As GSSOC'23 Contributor would like to contribute for this. Kindly,assign this to me.

### PR DESCRIPTION
**Pull request by SnigdhaP2022**
Created README file and added necessary content which helps for contributors while previewing

Close #2514 

# Checklist:

- [✅] I have mentioned the issue number in my Pull Request.
- [✅] I have commented my code, particularly in hard-to-understand areas
- [✅] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by SnigdhaP2022** [comment]
@MohitGupta121  Kindly, review this.

File: Frontend-Projects/Electronic Gadget/images/Landing page.png
Comment (MohitGupta121) [inline Frontend-Projects/Electronic Gadget/images/Landing page.png:1]: Remove this remove from PR and use them through link, upload somewhere in cloud and fetch from there

File: Frontend-Projects/Electronic Gadget/images/cart.png
Comment (MohitGupta121) [inline Frontend-Projects/Electronic Gadget/images/cart.png:1]: Same for this

File: Frontend-Projects/Electronic Gadget/images/login.png
Comment (MohitGupta121) [inline Frontend-Projects/Electronic Gadget/images/login.png:1]: Same for this image

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**